desc/compute/_geometry.py

```diff
@@ -77,12 +77,29 @@ def _V_r_of_r(params, transforms, profiles, data, **kwargs):
     transforms={"grid": []},
     profiles=[],
     coordinates="r",
-    data=["sqrt(g)_r", "sqrt(g)"],
+    data=["sqrt(g)_r"],
 )
 def _V_rr_of_r(params, transforms, profiles, data, **kwargs):
-    data["V_rr(r)"] = surface_integrals(
-        transforms["grid"], data["sqrt(g)_r"] * jnp.sign(data["sqrt(g)"])
-    )
```

Comment:
Now that the volume Jacobian is enforced to be positive, I've removed `jnp.sign(data["sqrt(g)"])` from some places and considering the sign to be positive at the magnetic axis. I need to do this so that things are differentiable and limits well-defined.